spec/plsql/package_spec.rb

```diff
@@ -20,7 +20,7 @@
         END test_procedure;
       END;
     SQL
-
+    plsql.execute("grant execute on TEST_PACKAGE to #{DATABASE_USERS_AND_PASSWORDS[1][0]}")
```

Comment:
That got me thinking about an edge case - what if we connect through [proxy user](http://www.pythian.com/blog/the-power-of-the-oracle-database-proxy-authenticated-connections/)? (I'm not sure whether the other tests would be compatible with such scenario either).